pkg/grpc/api.go

```diff
@@ -42,6 +42,7 @@ const (
 
 // API is the gRPC interface for the Dapr gRPC API. It implements both the internal and external proto definitions.
 type API interface {
+	channel.SetAppChannel
 	// DaprInternal Service methods
```

Comment:
We need to expose `SetActorRuntime(actors actors.Actors)` method because each API struct has actor runtime instance. however,  initActor is called after app is ready. so we need to call this method after initActor.

Reply:
Done, method name is `SetActor` hope that is fine?